projects/angular-sdk/src/lib/services/descope-auth.service.ts

```diff
@@ -137,6 +137,26 @@ export class DescopeAuthService {
 		return '';
 	}
 
+	isSessionTokenExpired(token = this.getSessionToken()) {
+		if (isBrowser()) {
+			return this.descopeSdk.isJwtExpired(token ?? '');
+		}
+		console.warn('Is session token expired is not supported in SSR');
+		return true;
+	}
+
+	isRefreshTokenExpired(token = this.getRefreshToken()) {
+		if (isBrowser()) {
+			return (
+				this.descopeSdk as AngularDescopeSDK & {
+					isJwtExpired: (token: string) => boolean | null;
+				}
+			).isJwtExpired(token ?? '');
+		}
+		console.warn('Is refresh token expired is not supported in SSR');
```

Comment:
I knot its like this in other funcitons, but I think the name of the function is better then an "english" name of it

```suggestion
		console.warn('"isRefreshTokenExpired" is not supported in SSR');
```